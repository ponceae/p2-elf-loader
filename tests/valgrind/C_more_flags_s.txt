==298494== Memcheck, a memory error detector
==298494== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==298494== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==298494== Command: ../y86 -s inputs/moreflags.o
==298494== 
 Segment   Offset    VirtAddr  FileSize  Type      Flag
  00       0x0060    0x0100    0x0015    DATA       W 
  01       0x0075    0x0200    0x000b    DATA        X
  02       0x0080    0x0300    0x0015    DATA       WX
  03       0x0090    0x0200    0x000b    DATA      RWX
==298494== 
==298494== HEAP SUMMARY:
==298494==     in use at exit: 0 bytes in 0 blocks
==298494==   total heap usage: 4 allocs, 4 frees, 20,952 bytes allocated
==298494== 
==298494== All heap blocks were freed -- no leaks are possible
==298494== 
==298494== For lists of detected and suppressed errors, rerun with: -s
==298494== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
