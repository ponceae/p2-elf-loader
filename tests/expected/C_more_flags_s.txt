 Segment   Offset    VirtAddr  FileSize  Type      Flag
  00       0x0060    0x0100    0x0015    DATA       W 
  01       0x0075    0x0200    0x000b    DATA        X
  02       0x0080    0x0300    0x0015    DATA       WX
  03       0x0090    0x0200    0x000b    DATA      RWX
